Shared/Samples/Get elevation at point on surface/GetElevationAtPointOnSurfaceView.swift

```diff
@@ -0,0 +1,131 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct GetElevationAtPointOnSurfaceView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The location callout placement.
+    @State private var calloutPlacement: CalloutPlacement?
+    
+    /// The screen point where to get the base surface location.
+    @State private var screenPoint: CGPoint!
+    
+    /// The scene point where the scene was tapped.
+    @State private var scenePoint: Point!
+    
+    /// The surface elevation of the tapped point.
+    @State private var elevation: Double?
+    
+    var body: some View {
+        SceneViewReader { sceneViewProxy in
+            SceneView(scene: model.scene)
+                .onSingleTapGesture { screenPoint, scenePoint in
+                    self.screenPoint = screenPoint
+                    self.scenePoint = scenePoint
+                }
+                .callout(placement: $calloutPlacement.animation(.default.speed(2))) { _ in
+                    VStack(alignment: .leading) {
+                        Text("Elevation")
+                            .font(.headline)
+                        Text(model.elevationString(elevation: elevation))
+                            .font(.callout)
+                    }
+                    .padding(5)
+                }
+                .task(id: scenePoint) {
+                    guard let scenePoint,
+                          // Converts the tapped screen point into a point on the surface.
+                          let relativeSurfacePoint = sceneViewProxy.baseSurfaceLocation(fromScreenPoint: screenPoint) else { return }
```

Comment:
You may wonder, why not just get the `z` value from the `scenePoint`? The design is intended to show the `baseSurfaceLocation` and `elevation(at:)` APIs, so the sample shows the *detour* to get the elevation.